### ISSUE TITLE
Safari (both Mac and iOS) are constantly crashing on SauseLabs with PR# 7279

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#7310]
Unrevert #7279 when fixed.

**Comment by aghassemi** [comment]
Caused by (now reverted) https://github.com/ampproject/amphtml/pull/7321

**Comment by aghassemi** [comment]
Blocked on https://github.com/ampproject/amphtml/issues/7822

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @aghassemi Do you have any updates?

**Comment by aghassemi** [comment]
Closing in favour of #10974
